File: MalESIEA-Airlines/MalEsieaAirline/src/main/java/Users/Billet.java
package Users;

import java.util.Random;

public class Billet {

    private String code_aller;
    private String code_retour;
    private int prix;
    private int id;

    public Billet (String code_aller, String code_retour, int prix){
        this.code_aller = code_aller;
        this.code_retour = code_retour;
        this.prix = prix;
        Random random = new Random();
        this.id = random.nextInt(1000);
    }

    public int getPrix() {
        return prix;
    }

    public void setPrix(int prix) {
        this.prix = prix;
    }

    public String getCode_aller() {
        return code_aller;
    }

    public void setCode_aller(String code_aller) {
        this.code_aller = code_aller;
    }

    public String getCode_retour() {
        return code_retour;
    }

    public void setCode_retour(String code_retour) {
        this.code_retour = code_retour;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        Random random = new Random();
        this.id = random.nextInt(1000);
    }

    /*
    public static void main(String[] args) {
        Billet billet = new Billet("NYC", "PAR", 800);
        String info = new String(billet.code_aller + "||" + billet.code_retour + "||" + billet.prix +"||" + billet.id);
        System.out.println(info);
    }
    */
}

File: FSI1-vingt-sur-vingt/projet/Solution-finale/app/Billet.php
<?php


namespace App;

Class Billet 
{
	public $_id;
	public $_CodeDepart;
	public $_CodeArriver;
	public $_Prix;


	
	public function __construct($_id,$_CodeDepart,$_CodeArriver,$_Prix)
	{
		$this->setId($_id);
		$this->setCodeDepart($_CodeDepart);
		$this->setCodeArriver($_CodeArriver);
		$this->setPrix($_Prix);
	}

	public function getId()
	{
		return $this->_id;
	}

	public function getCodeDepart()
	{
		return $this->_CodeDepart;
	}

	public function getCodeArriver()
	{
		return $this->_CodeArriver;
	}

	public function getPrix()
	{
		return $this->_Prix;
	}

	public function setId($_id)
	{
		return $this->_id = $_id;
	}

	public function setCodeDepart($_CodeDepart)
	{
		$this->_CodeDepart = $_CodeDepart;
	}


	public function setCodeArriver($_CodeArriver)
	{
		$this->_CodeArriver = $_CodeArriver;
	}

	public function setPrix($_Prix)
	{
		$this->_Prix = $_Prix;
	}
}







?>